sdk/compute/Microsoft.Azure.Management.Compute/src/Microsoft.Azure.Management.Compute.csproj

```diff
@@ -26,6 +26,7 @@
         9. Updates to the Install Patches API.
         10. Updates to the PatchStatus in VMInstanceView, and Windows and Linux patch settings. 
         11. Added the OrchestrationMode property to VMSS resources. 
+		12. Cloud Services Extended Support
```

Comment:
For a new release, do not just add it to the existing changelog. Erase the previous changelog and add your list. Also, the below Note has to be removed.

Reply:
sure let me make the change

Reply:
@Sandido Made the changes . Please review